_data/sites.json

```diff
@@ -8383,11 +8383,21 @@
     },
 
     {
+<<<<<<< HEAD
```

Comment:
Seems like you have unresolved merge lines here and on lines 8394, 8400.

Reply:
solved this!